### DISCUSSION TITLE
Font size not working

### DISCUSSION THREAD

**Discussion by mihaeldonko** [microweber/microweber#967] [Q&A]
Hello,

When i select text like this:

![image](https://user-images.githubusercontent.com/109346556/201620414-21a07433-c7d3-4d06-a13a-5bed0b9da83b.png)

The font size is not changing. Do i need to add certain class or something for it to work?

**Comment by peter-mw** [comment]
Hello, this is fixed in the dev branch and will be released in version 1.3.2
